provider/aws/dist/rack.json

```diff
@@ -971,6 +971,7 @@
                     "ecs:DiscoverPollEndpoint",
                     "ecs:Poll",
                     "ecs:RegisterContainerInstance",
+                    "ecs:StartTask",
```

Comment:
Not needed for the new approach of deploying the agent as a task. It feels right in general that an instance can manage its own tasks, but I'll remove it for sake of security.